src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.c

```diff
@@ -6,26 +6,138 @@
 
 #include "z_obj_makeoshihiki.h"
 
+#include "overlays/actors/ovl_Obj_Oshihiki/z_obj_oshihiki.h"
+
+#include <vt.h>
+
 #define FLAGS 0x00000020
 
 #define THIS ((ObjMakeoshihiki*)thisx)
 
 void ObjMakeoshihiki_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjMakeoshihiki_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
 const ActorInit Obj_Makeoshihiki_InitVars = {
-    ACTOR_OBJ_MAKEOSHIHIKI,
-    ACTORTYPE_PROP,
-    FLAGS,
-    OBJECT_GAMEPLAY_DANGEON_KEEP,
-    sizeof(ObjMakeoshihiki),
-    (ActorFunc)ObjMakeoshihiki_Init,
-    (ActorFunc)Actor_Noop,
-    (ActorFunc)Actor_Noop,
-    (ActorFunc)ObjMakeoshihiki_Draw,
+    ACTOR_OBJ_MAKEOSHIHIKI,       ACTORTYPE_PROP,          FLAGS,
+    OBJECT_GAMEPLAY_DANGEON_KEEP, sizeof(ObjMakeoshihiki), (ActorFunc)ObjMakeoshihiki_Init,
+    (ActorFunc)Actor_Noop,        (ActorFunc)Actor_Noop,   (ActorFunc)ObjMakeoshihiki_Draw,
```

Comment:
formatting is unfortunate here. did it not do it before because it was a comment?

Reply:
Yeah, I think so. And I agree...I wish it was more reasonable.